### ISSUE TITLE
Export menu button does not have a tooltip when accessibility module is attached

### ISSUE THREAD

**Issue by mfilipiec** [highcharts/highcharts#16307]
#### Expected behaviour
After hover on the export menu button should show a native tooltip with text: `Chart context menu` even when the accessibility module was attached.

#### Actual behaviour
When a module with accessibility is attached, the button has no native tooltip after hover on it. 

#### Live demo with steps to reproduce
https://jsfiddle.net/BlackLabel/9px0gsan/

Workaround: https://jsfiddle.net/BlackLabel/orwcmxpu/

#### Product version
v9.2.2

**Comment by billsaysthis** [comment]
Any progress here? We definitely need this fix.

**Comment by oysteinmoseng** [comment]
@billsaysthis Hi, the issue has been fixed and should be part of the next release. In the meantime, you should be able to use the workaround posted above for the same effect: https://jsfiddle.net/BlackLabel/orwcmxpu/
